### PR TITLE
[DOP-19931] Add manage_superusers script

### PR DESCRIPTION
**Pull request by IlyasDevelopment**
## Change Summary
Implemented a script to update the role of specific users in the database to superuser. Now list of superusers can be passed by `SYNCMASTER__ENTRYPOINT__SUPERUSERS` via docker-compose env variable.

## Checklist

* [x] Commit message and PR title is comprehensive
* [x] Keep the change as small as possible
* [x] Unit and integration tests for the changes exist
* [ ] Tests pass on CI and coverage does not decrease
* [ ] Documentation reflects the changes where applicable
* [x] `docs/changelog/next_release/<pull request or issue id>.<change type>.rst` file added describing change
  (see [CONTRIBUTING.rst](https://github.com/MobileTeleSystems/syncmaster/blob/develop/CONTRIBUTING.rst) for details.)
* [x] My PR is ready to review.

### CODE REVIEW

File: tests/test_database/test_manage_superusers.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    assert set(expected_not_superusers) == {user.username for user in not_superusers}
+    for user in not_superusers:
+        assert not user.is_superuser
+
+
+@pytest.mark.parametrize("simple_users", [10], indirect=True)
+async def test_remove_superusers(caplog, session: AsyncSession, simple_users: list[MockUser]):
+    # users 0 and 1 will be superusers, 2-10 will not
+    to_create = [user.username for user in simple_users[:5]]
```
Comment (dolfinus) [inline tests/test_database/test_manage_superusers.py:54]: Comment says about users 2-10, but here used only 2-5

File: docker-compose.yml [hunk 2]
```diff
@@ -36,9 +36,11 @@ services:
       context: .
     ports:
       - 8000:8000
-    # PROMETHEUS_MULTIPROC_DIR is required for multiple workers, see:
-    # https://prometheus.github.io/client_python/multiprocess/
     environment:
+      # list here usernames which should be assigned SUPERUSER role on application start
+      SYNCMASTER__ENTRYPOINT__SUPERUSERS: syncmaster
```
Comment (dolfinus) [inline docker-compose.yml:41]:
```suggestion
      SYNCMASTER__ENTRYPOINT__SUPERUSERS: admin
```

IMHO this makes more sense

File: syncmaster/backend/scripts/manage_superusers.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+async def list_superusers(session: AsyncSession) -> None:
+    result = await session.execute(select(User).filter_by(is_superuser=True).order_by(User.username))
+    superusers = result.scalars().all()
+    logging.info("Listing users with SUPERUSER role:")
+    for superuser in superusers:
+        logging.info("    %r", superuser.username)
+    logging.info("Done.")
+
+
+def create_parser() -> argparse.ArgumentParser:
```
Comment (dolfinus) [inline syncmaster/backend/scripts/manage_superusers.py:68]: Should we add this script usage example to documentation, or this is a part of some dedicated PR?

Comment (IlyasDevelopment) [reply syncmaster/backend/scripts/manage_superusers.py:68]: Added usage example for docker, we can also extend script usage later

**Review by dolfinus — APPROVED** [review]

**Pull request merged**